ixofoundation/ixo-webclient
# feat: create basic text input generator

## Scope
https://nonaredteam.atlassian.net/browse/IXO-821

## Work Done
Added Toolbar for setting whether the field is required as well as duplicating and deleting the field
Added Redux for adding adding, removing and duplicating fields

## Steps to Test
Make sure you can add a short text field and enter the details and the details save to the redux store

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/88937042-e30c3c00-d283-11ea-83c4-40139ff8ea0a.png)
